test/src/test/java/jenkins/util/JenkinsJVMRealTest.java

```diff
@@ -17,7 +17,7 @@
     public static JenkinsRule j = new JenkinsRule();
 
     @Test
-    public void isJenkinsJVM() throws Exception {
+    public void isJenkinsJVM() throws Throwable {
```

Comment:
@oleg-nenashev I tried to separate each "fix" in a dedicated commit. This is https://github.com/jenkinsci/jenkins/pull/2802/commits/77dfa64b6a173b58c7f61107dcc51a61f7a3466a